### DISCUSSION TITLE
Importing frequencies from SDR#

### DISCUSSION THREAD

**Discussion by Y-Perry** [AlexandreRouma/SDRPlusPlus#314] [Q&A]
I have a ton of identified frequencies in SDR#, stored in an XML file. How can I import them into SDRPP format of JSON file?

Thanks

Yonadav

**Comment by AlexandreRouma** [comment]
Hello,
Short answer: you can't,
long asnwer: The formats are completely different and I've never used the SDR# one.
Send your file and I'll see if I can convert it for you

**Comment by Y-Perry** [comment]
Thanks, Alexandre.

Attached is my SDR# XML file.

Yonadav

**Reply by the-gobleo-tinkerer** [reply]
In looks like your attachment isn't coming through via email. You might need to post it here in your browser.

**Comment by Y-Perry** [comment]
This is not a file that I want to post public. Please give me a private
email address that I can send it to.

Thanks,

Yonadav
